network-api/networkapi/wagtailpages/pagemodels/customblocks/datawrapper_block.py

```diff
@@ -0,0 +1,64 @@
+from wagtail.embeds import blocks
+
+
+class DatawrapperBlock(blocks.EmbedBlock):
+    """
+    Embed block for Datawrapper visualisations.
+
+    Currently, there is no limitation in the block itself on which oEmbed sources
```

Comment:
is oEmbed the technical term, or is it a shorthand? (if the latter, let's use the full term)

Reply:
It appears to be the technical term: https://github.com/iamcal/oembed